docs/application/toc_all.md

```diff
@@ -709,6 +711,7 @@
 #### [Running and Debugging Applications](/application/web/tutorials/process/run-debug-app.md)
 ### [Application Filtering](/application/web/tutorials/app-filtering.md)
 ### [Security and API Privileges](/application/web/tutorials/sec-privileges.md)
+### [Security and API Privileges for Old Apps](/application/web/tutorials/old-versioned-sec-privileges.md)
```

Comment:
Suggestion: Old apps does not convey the right meaning. It will be good to specify the version.

Suggest rephrasing to:

[Security and API Privileges for Apps with API Version 4.0 or Earlier](old-versioned-sec-privileges.md)

or

[Security and API Privileges for 4.0 Apps or Earlier](old-versioned-sec-privileges.md)

Please verify and update.

Reply:
done